README.md

```diff
@@ -1,83 +1,25 @@
-# {{GROUP_NAME}} {{GROUP_TYPE}} Group
-<!--
- This is the template for creating project groups in rust-lang. Be sure to go
- through all sections marked with `**FIX ME**`, and make sure that the text is
- correct, and feel free to replace/remove any part that's not relevant to
- your group.
+# Error Handling Project Group
 
- All of the text across all of the initial files uses the same group of
- variables to allow for easy search and replace. They are listed below.
+Welcome to the repository for the Error Handling Project Group! We're focused on  defining and communicating Rust's error handling story.
 
- Example sed command: `sed -i '' 's/{{GROUP_NAME}}/Inline ASM/g' ./**/*.md`
- *Note* the `-i ''` is important as it is required on some platforms e.g. macOS
-
- * {{GROUP_NAME}} -> The display name of your group e.g. "Inline ASM".
- * {{GROUP_SLUG}} -> The url slug name of your group used for
-   `rust-lang/team` and repo name. e.g. "pg-inline-asm".
- * {{CHAT_PLATFORM}} -> The name of your chat app e.g. "Zulip".
- * {{CHAT_LINK}} -> The hyperlink to your discussions on the chat app
-   e.g. "https://rust-lang.zulipchat.com/#narrow/stream/216763-project-inline-asm".
--->
-
-<!--
- Status badge advertising the project as being actively worked on. When the
- project has finished be sure to replace the active badge with a badge
- like: https://img.shields.io/badge/status-archived-grey.svg
--->
-![project group status: active](https://img.shields.io/badge/status-active-brightgreen.svg)
-[![project group documentation](https://img.shields.io/badge/MDBook-View%20Documentation-blue)][gh-pages]
-
-
-**FIX ME**
-
-<!--
- Provide a short introduction about your project group. Make sure to include any
- relevant links to information about your group.
--->
-
-Welcome the repository for the {{GROUP_NAME}} Project Group! This is the
-repository we use to organise our work. Please refer to our [charter] as well
-as our [github pages website][gh-pages] for more information on our goals and
+This is the repository we use to organise our work. Please refer to our [charter] for more information on our goals and
 current scope.
 
-[charter]: ./CHARTER.md
-[gh-pages]: https://rust-lang.github.io/{{GROUP_SLUG}}
-
-**/FIX ME**
+ - Shepherd:
+    - [Jane Lusby (yaahc)](https://github.com/yaahc)
+    - [Sean Chen (seanchen1991)](https://github.com/seanchen1991)
+ - Rust library team contact:
+    - [Andrew Gallant (BurntSushi)](https://github.com/burntsushi)
```

Comment:
I'm not positive that @BurntSushi is acting as the library team contact, I got the impression that he wanted to participate more as a member, so I want to get his eyes on this before it merges.

Reply:
Yeah thank you for calling this out. Basically, I don't have the bandwidth to participate in this effort in anything more than a passive/advisory capacity. (I have a new baby arriving in less than a month.) So I think having me be responsible for shepherding this would be a bad idea. :-(

Reply:
Should I leave you on here as the lib team contact, or put you down under an "Advisor" category?

Reply:
> I have a new baby arriving in less than a month.

So exciting! 😁

I'll be happy to volunteer. We're still working out how we want these group to operate within Libs so am not totally sure what we expect the contact to do yet but we'll figure that out!